thrift/shared.thrift

```diff
@@ -1808,12 +1808,24 @@ struct CrossClusterRecordChildWorkflowExecutionCompleteRequestAttributes {
 struct CrossClusterRecordChildWorkflowExecutionCompleteResponseAttributes {
 }
 
+struct CrossClusterApplyParentClosePolicyRequestAttributes {
```

Comment:
looks like this attribute is the information for applying parent close policy for one workflow. Does it mean we will generate N x-cluster tasks for N childworkflows in the source cluster, even if their target cluster is the same. Or there will be only one x-cluster task, but when sending the request to the target cluster, we represent them using N attributes?